3drepo/3drepo.io
# Issue 3626 - Custom ticket comments (no images)

This fixes #3626

#### Description
Inside custom tickets it is now possible to add comments


#### Test cases
The comments-panel was added to custom tickets (make sure the template allows comments)
- Create tickets
- Reply to tickets
- Delete tickets
- Edit tickets
